sema4ai/tests/sema4ai_code_tests/test_vscode_integration.py

```diff
@@ -2319,7 +2319,7 @@ def test_agent_cli_version(
     assert result["result"] is not None
 
 
-def test_create_agent_package(
+def test_create_and_pack_agent_package(
```

Comment:
Humm, the test was renamed, but it seems it's missing the code that'd check for the agent pack.

Reply:
Yes, will add it. I changed the name to remember where to do the test.